File: static/js/UGPrint.js
let url = d3.select("#url");
let submit = d3.select("#submit");
let tabview = d3.select("#tab");
let status = d3.select("#status");
status.style("opacity", 0);

submit.on("click", () => {
    status.style("opacity", 100);
    d3.text("/getTab", {
        method: "POST",
        headers: { "Content-Type": "application/x-www-form-urlencoded" },
        body: "url=" + JSON.stringify(url.node().value)
    }).then(data => {
        status.style("opacity", 0);
        tabview.text(data);
    });
});

File: UGPrint.py
import requests
from bs4 import BeautifulSoup as bs
import json
from flask import Flask,render_template, request

app = Flask(__name__)

def get_tab(tab_url):
    try:
        code = requests.get(tab_url)
        soup = bs(code.text, 'html.parser')

        js = soup.find('div', class_='js-store').attrs['data-content']
        content = json.loads(js)
        tab = content['store']['page']["data"]["tab_view"]['wiki_tab']['content']

        tab = tab.replace('[tab]', '').replace(
            '[/tab]', '').replace('[ch]', '').replace('[/ch]', '')
    except:
        return "Error Loading Tab"
    return tab

@app.route('/')
def index():
    return render_template('index.html', content=None)

@app.route('/getTab', methods=['POST'])
def get_tab_route():
    if request.method == 'POST':
        url = request.form['url']
        return get_tab(url.replace('"',''))

if __name__ == '__main__':
    app.run(debug=True)

